graylog2-server/src/main/java/org/graylog/plugins/views/search/rest/scriptingapi/mapping/AggregationTabularResponseCreator.java

```diff
@@ -96,6 +114,30 @@ private static List<List<Object>> getDatarows(final AggregationRequestSpec searc
                 .collect(Collectors.toList());
     }
 
+    private Stream<Object> decorateGroupings(ImmutableList<String> keys, AggregationRequestSpec searchRequestSpec, Set<FieldDecorator> decorators, SearchUser searchUser) {
+        List<Object> result = new ArrayList<>();
+        for(int i = 0; i < keys.size(); i++) {
+            final String value = keys.get(i);
+            final RequestedField field = searchRequestSpec.groupings().get(i).requestedField();
+            final Object decorated = decorate(decorators, field, value, searchUser);
+            result.add(decorated);
+        }
+        return result.stream();
+    }
+
+    private Object decorate(Set<FieldDecorator> decorators, RequestedField field, Object val, SearchUser searchUser) {
+        final Optional<Object> decorated = decorators.stream()
+                .filter(d -> d.accept(field))
+                .findFirst()
```

Comment:
Is there any chance that at any point we will have conflicting decorators, so that single field is accepted by multiple decorators?
If yes, is it ok that we pick the first one and silently ignore the rest?
If no, can we somehow make it more explicit that we do not expect more than one decorator for one field?

Reply:
There is one problem with this approach: the wrong decorator setup will be identified in the unknown moment in the future, when someone invokes a call for certain field and decorator.
I'd prefer to know it straight away, during the startup.

I just wonder if we can simplify "accept" logic, so that wrong setup is identified when decorators are bound in the container?
We can talk more about it if you want.